tests/sentry/lang/javascript/test_processor.py

```diff
@@ -94,6 +96,108 @@ def test_missing_dist(self, _1, _2):
+                                    "function": "?",
+                                    "lineno": 4,
+                                    "colno": 0,
+                                },
+                            ]
+                        },
+                    }
+                ]
+            },
+        }
+
+        stacktrace_infos = [
+            stacktrace for stacktrace in find_stacktraces_in_data(data, with_exceptions=True)
+        ]
+        processor = JavaScriptStacktraceProcessor(
+            data={"release": release.version, "dist": "foo", "timestamp": 123.4},
+            project=project,
+            stacktrace_infos=stacktrace_infos,
+        )
+
+        frames = processor.get_valid_frames()
+        assert processor.suspected_console_errors(frames) is True
+
+        processor.tag_suspected_console_errors(frames)
+        assert get_tag(processor.data, "empty_stacktrace.js_console") is True
+
+    def test_no_suspected_console_error(self):
+        project = self.create_project()
+        release = self.create_release(project=project, version="12.31.12")
+
+        data = {
+            "is_exception": True,
+            "platform": "javascript",
+            "project": project.id,
+            "exception": {
+                "values": [
+                    {
+                        "type": "SyntaxError",
+                        "mechanism": {
+                            "type": "onerror",
+                        },
+                        "value": ("value"),
+                        "stacktrace": {
+                            "frames": [
+                                {
+                                    "abs_path": "http://example.com/foo.js",
+                                    "filename": "<anonymous>",
+                                    "function": "name",
+                                    "lineno": 4,
+                                    "colno": 0,
+                                },
+                                {
+                                    "abs_path": "http://example.com/foo.js",
+                                    "filename": "<anonymous>",
+                                    "function": "new name",
+                                    "lineno": 4,
+                                    "colno": 0,
+                                },
+                            ]
+                        },
+                    }
+                ]
+            },
+        }
+
+        stacktrace_infos = [
+            stacktrace for stacktrace in find_stacktraces_in_data(data, with_exceptions=True)
+        ]
+
+        processor = JavaScriptStacktraceProcessor(
+            data={"release": release.version, "dist": "foo", "timestamp": 123.4},
+            project=project,
+            stacktrace_infos=stacktrace_infos,
+        )
+
+        frames = processor.get_valid_frames()
+        assert processor.suspected_console_errors(frames) is False
```

Comment:
Can I assume that we're trying to be safe and ignore anything that has 2 frames or more? (Since we're returning False)